console/src/main/webapp/manager/app/components/user/user.es6

```diff
@@ -28,6 +27,19 @@ class UserController {
       } else {
         user.validOrg = true
       }
+      if (this.tab === 'delegations') {
```

Comment:
This part of code will be executed only if user select delegation tab ?

Reply:
yep